src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+
 body {
   margin: 0;
+  font-family: "Roboto", "Arial", sans-serif;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  justify-content: space-between;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    color: #616070;
+    padding-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
it's default value, so we can remove it

Reply:
if you remove these lines, the text becomes many times larger and then the check does not pass.